---
title: C6103
ms.date: 11/04/2016
ms.technology: vs-ide-code-analysis
ms.topic: reference
f1_keywords:
- C6103
ms.assetid: 22d1ab35-31a3-4ba9-8ef4-7a64bce66621
author: mikeblome
ms.author: mblome
manager: wpickett
ms.workload:
- multiple
ms.openlocfilehash: 6dd670cc6549ec96cea56168599395cb95df599f
ms.sourcegitcommit: 42ea834b446ac65c679fa1043f853bea5f1c9c95
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 04/19/2018
---
# <a name="c6103"></a>C6103
avviso C6103: restituzione \<variabile > Impossibile chiamata di funzione alla riga dal \<percorso >.

 Un percorso corretto tramite la funzione restituisce una variabile utilizzata come parametro `_Out_` a una chiamata di funzione interna non riuscita.

 Il problema potrebbe essere causato dal fatto che la funzione chiamata e la funzione chiamante non sono completamente annotate. La funzione chiamata può richiedere `_Always_`, `_Outptr_result_nullonfailure_` (`_COM_Outptr_` per codice COM) o un'annotazione correlata e la funzione chiamante può richiedere un'annotazione `_Success_`. Un'altra causa possibile del problema consiste nel fatto che l'annotazione `_Success_` nella funzione chiamata non è corretta.

## <a name="see-also"></a>Vedere anche
 [Uso delle annotazioni SAL per ridurre gli errori del codice C/C++](../code-quality/using-sal-annotations-to-reduce-c-cpp-code-defects.md)